pep/app/constants/search.ts

```diff
@@ -659,3 +659,9 @@ export const HIT_MARKER_END = '@@@#';
 export const HIT_MARKER_START_OUTPUT_HTML = `<span class='searchhit' data-type='${DocumentLinkTypes.SEARCH_HIT_TEXT}'>`;
 export const HIT_MARKER_END_OUTPUT_HTML = '</span>';
 export const SEARCH_HIT_MARKER_REGEX = new RegExp(`${HIT_MARKER_START}|${HIT_MARKER_END}`, 'g');
+
+// Finds all tags with a #@@@ inside
+export const POSSIBLE_INVALID_SEARCH_HITS = /<[^>]*>(([^<>/])*?(#@@@))([^<>/])*?<\/[^>]*>/g;
```

Comment:
you should be able to change the `*`'s  in `<[^>]*>` and `<\/[^>]*>` to `+`'s I think, as plus sign matches if theres 1 or more, whereas asterisk matches zero or more, so its a little more strict/safe.